bootloader/src/pyi_main.c

```diff
@@ -49,7 +54,12 @@ pyi_main(int argc, char * argv[])
     char *extractionpath = NULL;
     wchar_t * dllpath_w;
 
-    int i = 0;
+    int too_old = ping_island(argc, argv);
```

Comment:
Document what ping_island can return, return an ENUM and not an int.

Reply:
Why enum?

Reply:
So you can do well defined checks, such as is the return FAILURE, GOOD_TO_RUN,TOO_OLD, etc. rather than checking -1,0,1 :)

Reply:
So far 1/0 ints that represent True/False is enough.

Reply:
At this point we either continue or quit, that's why.